src/rulesets/Base/base.jl

```diff
@@ -119,6 +119,40 @@ function rrule(::typeof(hypot), z::Complex)
     return (Ω, hypot_pullback)
 end
 
+# Note that `hypot` with two arguments has rules in `fastmath_able.jl`
+
+function frule(
+    (_, Δx, Δy, Δz, Δxs...),
+    ::typeof(hypot),
+    x::T,
+    y::T,
+    z::T,
+    xs::Vararg{T, N}
+) where {T<:Union{Real,Complex}, N}
```

Comment:
It's probably not necessary that all arguments are of the same type?
```suggestion
    x::Union{Real,Complex},
    y::Union{Real,Complex},
    z::Union{Real,Complex},
    xs::Union{Real,Complex}...,
)
```